.github/workflows/start-release.yml

```diff
@@ -0,0 +1,53 @@
+name: Trigger Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release-setup:
+    name: Setup Release Branch
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v3
+      id: checkout
+      with:
+        submodules: "recursive"
+        ref: "develop"
+    - name: Pull Tags
+      id: pull-tags
+      # We don't want a full heavyweight checkout with full history
+      # checkout action only can get tags + full history, so do this separately.
+      # Don't need tags in submodules.
+      run: git fetch --no-recurse-submodules origin 'refs/tags/*:refs/tags/*'
```

Comment:
what happens if we run `actions/checkout` with `ref: refs/tags/*`?

Reply:
Doesn't have the intended effect, and the docs specifically mention that to checkout tags, you need depth=0 (checkout full history + everything)